### PR TITLE
Feature/Update to swift 2.1

### PR DESCRIPTION
**Pull request by justinvallely**

### CODE REVIEW

File: Pod/Classes/SwiftPhotoGallery.swift [hunk 1]
```diff
@@ -73,7 +73,7 @@ public class SwiftPhotoGallery: UIViewController, UICollectionViewDataSource, UI
         self.delegate = delegate
     }
 
-    required public init(coder aDecoder: NSCoder) {
+    required public init?(coder aDecoder: NSCoder) {
```
Comment (LunaCodeGirl) [inline Pod/Classes/SwiftPhotoGallery.swift:76]: Why'd this change? Was the UIViewController a failable initializer?

File: Pod/Classes/SwiftPhotoGallery.swift [hunk 2]
```diff
@@ -85,7 +85,7 @@ public class SwiftPhotoGallery: UIViewController, UICollectionViewDataSource, UI
 
         } else {
 
-            var indexPaths: [NSIndexPath] = imageIndexes.map({NSIndexPath(forItem: $0, inSection: 0)})
+            let indexPaths: [NSIndexPath] = imageIndexes.map({NSIndexPath(forItem: $0, inSection: 0)})
```
Comment (LunaCodeGirl) [inline Pod/Classes/SwiftPhotoGallery.swift:88]: Good call

File: Pod/Classes/SwiftPhotoGallery.swift (hunk 1 above)
Comment (justinvallely) [inline Pod/Classes/SwiftPhotoGallery.swift:76]: Not sure. That's an auto-update change from Xcode.

File: Pod/Classes/SwiftPhotoGallery.swift (hunk 2 above)
Comment (justinvallely) [inline Pod/Classes/SwiftPhotoGallery.swift:88]: Thanks! (auto-update though)

**Comment by LunaCodeGirl** [comment]
Looks good to me! 👍

**Pull request merged**